design/pv_backup_info.md

```diff
@@ -82,12 +84,20 @@ type VeleroNativeSnapshotInfo struct {
 
 // PodVolumeBackupInfo is used for displaying the PodVolumeBackup snapshot status.
 type PodVolumeBackupInfo struct {
-    SnapshotHandle      string       // The actual snapshot ID. It can be the cloud provider's snapshot ID or the file-system uploader's snapshot.
+    SnapshotHandle      string       // The actual snapshot ID. It's the file-system uploader's snapshot ID for PodVolumeBackup.
     Size                int64        // The snapshot corresponding volume size. Some of the volume backup methods cannot retrieve the data by current design, for example, the Velero native snapshot.
 
     UploaderType  string    // The type of the uploader that uploads the data. The valid values are `kopia` and `restic`. It's useful for file-system backup and snapshot data mover.
-    VolumeName    string // The PVC's corresponding volume name used by Pod
-    PodName       string // The Pod name mounting this PVC. The format should be <namespace-name>/<pod-name>.
+    VolumeName    string   // The PVC's corresponding volume name used by Pod
```

Comment:
Is it PVC name? Or the mount path of the PVC in the Pod, or perhaps PV name? It is not clear just from the comment. Can you please clarify?

Reply:
No, it's the volume name in the PodSpec. It's used for displaying PVB information of backup.
https://github.com/kubernetes/kubernetes/blob/e4b74dd12fa8cb63c174091d5536a10b8ec19d34/pkg/apis/core/types.go#L48